documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -44,533 +48,515 @@ Available cameras
+For `rpicam-vid`, disables extra colour-denoise processing.
 
-The `--mode` option affects the mode choice for video recording and stills capture. To control the mode choice during the preview phase prior to stills capture, use the `--viewfinder-mode` option.
 
-----
-	--viewfinder-mode		Specify sensor mode, given as <width>:<height>:<bit-depth>:<packing>
-----
+Useful for image analysis when combined with xref:camera_software.adoc#post-processing[image post-processing].
 
-This option is identical to the `--mode` option except that it applies only during the preview phase of stills capture (also used by the `rpicam-hello` application).
+==== `hflip`
 
-----
-	--lores-width			Low-resolution image width <width>
-	--lores-height			Low-resolution image height <height>
-----
+Flips the image horizontally. Does not accept a value.
 
-`libcamera` allows the possibility of delivering a second, lower-resolution image stream from the camera system to the application. This stream is available in both the preview and the video modes (i.e. `rpicam-hello` and the preview phase of `rpicam-still`, and `rpicam-vid`), and can be used, among other things, for image analysis. For stills captures, the low-resolution image stream is not available.
+==== `vflip`
 
-The low-resolution stream has the same field of view as the other image streams. If a different aspect ratio is specified for the low-resolution stream, then those images will be squashed so that the pixels are no longer square.
+Flips the image vertically. Does not accept a value.
 
-During video recording (`rpicam-vid`), specifying a low-resolution stream will disable some extra colour-denoise processing that would normally occur.
+==== `rotation`
 
-Example: `rpicam-hello --lores-width 224 --lores-height 224`
+Rotates the image extracted from the sensor. Accepts only the values 0 or 180.
 
-Note that the low-resolution stream is not particularly useful unless used in conjunction with xref:camera_software.adoc#post-processing[image post-processing].
+==== `roi`
 
-----
-	--hflip				Read out with horizontal mirror
-	--vflip				Read out with vertical flip
-	--rotation			Use hflip and vflip to create the given rotation <angle>
-----
+Crops the image extracted from the full field of the sensor. Accepts a proportion of the available field of view in the following format: `<x>,<y>,<w>,h>`
 
-These options affect the order of read-out from the sensor, and can be used to mirror the image horizontally, and/or flip it vertically. The `--rotation` option permits only the value 0 or 180, so note that 90 or 270 degree rotations are not supported. Moreover, `--rotation 180` is identical to `--hflip --vflip`.
+These value define the following proportions:
 
-Example: `rpicam-hello --vflip --hflip`
+* `<x>`: X coordinates to skip before extracting an image
```

Comment:
IMHO these descriptions need to make clearer that they're a fraction of the complete image width (in the range 0 - 1), and not a count of pixels.